nginx-sys/src/lib.rs

```diff
@@ -75,19 +75,50 @@ pub unsafe fn str_to_uchar(pool: *mut ngx_pool_t, data: &str) -> *mut u_char {
 }
 
 impl ngx_str_t {
+    /// Returns the contents of this `ngx_str_t` as a byte slice.
+    ///
+    /// The returned slice will **not** contain the optional nul terminator that `ngx_str_t.data`
+    /// may have.
+    #[inline]
+    pub fn as_bytes(&self) -> &[u8] {
+        if self.is_empty() {
+            Default::default()
```

Comment:
Just curious, isn't `&[]` a more concise and explicit here? The same for the `as_bytes_mut()`.